translations/monero-core_sv.ts

```diff
@@ -1050,29 +1050,29 @@
     <message>
         <location filename="../pages/Settings.qml" line="240"/>
         <source>Bootstrap node</source>
-        <translation type="unfinished"></translation>
+        <translation>Startnod</translation>
```

Comment:
Bootstrap här syftar väl till sättet den startas på.... Så det syftar på en handling, medan din översättning är ett substantiv. Tillför verkligen Bootstrap någon information, eller kan man bara skriva "Starta nod".

Reply:
Nja, 'Bootstrap' här är väl inget verb som betyder starta? För det används ju ordet "Start" (start daemon, start mining, start local node etc)

Tittar man i ../pages/Settings.qml på rad 240 (eller kör programmet) så ser man att det är etiketten till ett inmatningsfält där man skriver in adress (och port) till en "Bootstrap node", dvs en nod som används för "bootstrapping". "bootstrap node" = substantiv alltså.

Sedan håller jag med om att "Startnod" inte förklarar vad det handlar om eftersom "bootstrap"-noden väl är en fjärrnod som används temporärt fram tills att den lokala noden har synkat klart, dvs det är ett sätt för plånboken att vara användbara snabbare, utan att behöva vänta på synk.

Men bootstrap kanske inte är bra på engelska heller, det kanske borde heta "temporary remote node" eller något. På spanska och franska har dom använt motsvarande "startnod" men de flesta andra språk tycks använda motsvarande "bootstrap-nod", så jag tror jag kör på det också.

Reply:
Ok, om det är substantiv här så har du rätt